modules/unauthenticated-users-system-scope-impersonation.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * authentication/tokens-scoping.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="unauthenticated-users-scope-impersonation_{context}"]
+= Adding unauthenticated users to the system:scope-impersonation role binding
+
+As a cluster administrator, you can add unauthenticated users to the `system:scope-impersonation` role binding in {product-title} for specific namespaces. The `system:scope-impersonation` role binding allows users to trigger builds from external systems that do not use an {product-title} authentication mechanism. Unauthenticated users do not have access to non-public role bindings by default. This is a change from {product-title} versions before 4.16.
```

Comment:
Same here, but with `system:scope impersonation` instead of `system:oauth-token-deleter`.
https://github.com/openshift/openshift-docs/pull/79148/files#r1765399432